source/containers.tex

```diff
@@ -5551,8 +5554,11 @@
 \remarks
 Causes reallocation if the new size is greater than the old capacity.
 Reallocation invalidates all the references, pointers, and iterators
-referring to the elements in the sequence.
-If no reallocation happens, all the iterators and references before the insertion point remain valid.
+referring to elements in the sequence, as well as the past-the-end iterator.
+If no reallocation happens, all references, pointers, and iterators
+referring to elements before the insertion point remain valid; all other
+references, pointers, and iterators referring to elements in the sequence,
+as well as the past-the-end iterator, are invalidated.
```

Comment:
Are the changes here justified by existing normative wording, or merely by this being the obvious (but unstated) intent of the existing rules?

Reply:
The addition of "as well as the past-the-end iterator" is justified by (1) harmonizing with the existing normative wording for "reallocation" given in `shrink_to_fit`, and (2) in practice, vendors do invalidate it.
The addition of "all other foos are invalidated" is justified by (1) assuming that this merely clarifies an implication-by-omission in the old wording, and (2) in practice, vendors do invalidate them.

IMHO this is good enough justification to count as "editorial," but obviously you'd know better than I would. :)